Osiris/Hacks/Ragebot.cpp

```diff
@@ -218,8 +218,11 @@ Vector GetHitBoxes(Entity* entity, Entity* weapon, int weaponIndex)noexcept
 		hitboxes.push_back(HitBoxes::HITBOX_PELVIS);
 	}
 
-
-	float minDamage = config->ragebot[weaponIndex].WallDamage;
+	float minDamage;
+	if (!config->ragebot[weaponIndex].DamageOverride || !GetAsyncKeyState(config->ragebot[weaponIndex].DamageOverrideKey))
+		minDamage = config->ragebot[weaponIndex].WallDamage;
+	else if (config->ragebot[weaponIndex].DamageOverride && GetAsyncKeyState(config->ragebot[weaponIndex].DamageOverrideKey))
+		minDamage = config->ragebot[weaponIndex].minOverrideDamage;
```

Comment:
Wouldn't it be better to name this `config->ragebot[weaponIndex].WallDamage` in GUI as "Min damage" instead of "Min wall damage"? It is kinda confusing as it suggests that this is the minimum damage to be done as wallbang for shot to be fired.

Reply:
emmmm OK